lib/jsdom/level1/core.js

```diff
@@ -245,7 +245,7 @@ core.DOMImplementation.prototype = {
 };
 
 core.Node = function Node(ownerDocument) {
-  EventTarget.call(this);
+  EventTarget.init.call(this);
```

Comment:
Why is init a method instead of a function? (i.e. why `.call(this)` instead of just `(this)`?)